### DISCUSSION TITLE
net::ERR_SSL_PROTOCOL_ERROR when connecting to supabase

### DISCUSSION THREAD

**Discussion by milestones95** [supabase/supabase#2279] [Questions]
Hi,

I keep getting this network protocol error when I connect to supabase api. Even through the UI when I want to see my tables. Has anyone encountered this issue before?

<img width="830" alt="Screen Shot 2021-07-08 at 7 17 15 AM" src="https://user-images.githubusercontent.com/47959617/124912948-8d283400-dfbc-11eb-9b29-de25ed82bae2.png">

**Comment by pranavbadami** [comment]
Hey team, some of our users are getting this issue when trying to sign in via 3rd party login (Google in our case). I didn't personally experience it on the dashboard like @milestones95 did tho. Can y'all help us narrow down the issue? Thanks!

![image](https://user-images.githubusercontent.com/6540790/125004073-8f66ae80-e00d-11eb-9aab-fce4271b534d.png)
